pkg/kfconfig/types.go

```diff
@@ -530,6 +551,54 @@ func (c *KfConfig) SyncCache() error {
 	return nil
 }
 
+func untar(body []byte, cacheDir string) error {
```

Comment:
golang has an untar library. Can this be used instead or reimplementing the function..
https://godoc.org/golang.org/x/build/internal/untar